docs/simple_switch.md

```diff
@@ -300,12 +302,13 @@ if (resubmit was called) {
     determined by the field list given as an argument to the last
     resubmit operation called.  The resubmitted packet will have
     instance_type equal to PKT_INSTANCE_TYPE_RESUBMIT.
-} else if (mcast_grp != 0) {
+}  else if (mcast_grp != 0) {
     // This condition will be true if your code made an assignment to
-    // standard_metadata.mcast_grp during ingress processing.  There
-    // are no special primitive actions built in to simple_switch for
-    // you to call to do this -- use a normal P4_16 assignment
-    // statement, or P4_14 modify_field() primitive action.
+    // `standard_metadata.mcast_grp`` during ingress processing AND did
```

Comment:
double backquote should probably be single backquote?  And another similar occurrence after modify_field() below.  Also seems like spurious addition of a space before "else if (mcast_grp != 0)" above, unless you think that is an improvement on the whitespace in the code.

Reply:
Indeed, seems like I was tired :) Should be fixed.